docs/static/breaking-changes.asciidoc

```diff
@@ -29,7 +30,47 @@ and have the persistent queue enabled, we strongly recommend that you drain or
 delete the persistent queue before you upgrade. See <<upgrading-logstash-pqs>>
 for information and instructions.
 
+
+[[breaking-7.0]]
+=== Breaking changes in 7.0
+
+Here are the breaking changes for 7.0. 
+
+[float]
+==== Changes in Logstash Core
+
+These changes can impact any instance of Logstash and are plugin agnostic, but only if you are using the features that are impacted.
+
+
+[float]
+==== Changes in Logstash Plugins
+
+Here are the changes to plugins.
```

Comment:
```suggestion
With 7.0.0, we have taken the opportunity to upgrade a number of bundled plugins to their newest major version, absorbing their breaking changes into the Logstash distribution.

While these upgrades included new features and important fixes, only the breaking changes are called out below.

NOTE: The majority of the changes to plugins are the removal of previously-deprecated and now-obsolete options; please be careful to ensure that your pipeline configurations do not use these removed options before upgrading.


===== Codec Plugins

*CEF Codec*

* Removed obsolete `sev` option
* Removed obsolete `deprecated_v1_fields` option

*Netflow Codec*

* Changed decoding of application_id to implement RFC6759; the format changes from a pair of colon-separated ids (e.g. `0:40567`) to a variable number of double-dot-separated ids (e.g. `0..12356..40567`).


===== Filter Plugins

====== Clone Filter

* Make `clones` a required option

====== Geoip Filter

* Removed obsolete `lru_cache_size` option

====== HTTP Filter

* Removed obsolete `ssl_certificate_verify` option


===== Input Plugins

====== Beats Input

* Removed obsolete `congestion_threshold` option
* Removed obsolete `target_field_for_codec` option
* Changed default value of `add_hostname` to false

NOTE: In Beats 7.0.0, the fields exported by Beats _to_ the Logstash Beats Input conform to the {ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the exported fields have been renamed, so you may need to modify your pipeline configurations to access them at their new locations prior to upgrading your Beats. See {beats-ref}/breaking-changes-7.0.html[Beats Breaking
changes in 7.0] for the full list of changed names.

====== HTTP Input

* Removed obsolete `ssl_certificate_verify` option

====== HTTP Poller Input

* Removed obsolete `interval` option
* Removed obsolete `ssl_certificate_verify` option

====== Tcp Input

* Removed obsolete `data_timeout` option
* Removed obsolete `ssl_cacert` option


===== Output Plugins

====== Elasticsearch Output

* Remove support for parent/child (still support join data type) since we don't support multiple document types any more
* Removed obsolete `flush_size` option
* Removed obsolete `idle_flush_time` option

====== HTTP Output

* Removed obsolete `ssl_certificate_verify` option

====== Kafka Output

* Removed obsolete `block_on_buffer_full` option
* Removed obsolete `ssl` option
* Removed obsolete `timeout_ms` option

====== Redis Output

* Removed obsolete `queue` option
* Removed obsolete `name` option

====== Sqs Output

* Removed obsolete `batch` option
* Removed obsolete `batch_timeout` option

====== Tcp Output

* Removed obsolete `message_format` option
```